Makefile

```diff
@@ -36,3 +37,15 @@ release: iso get_gh-release
 	cp $(BUILD_DIR)/minishift-b2d.iso release/
 	$(BUILD_DIR)/gh-release checksums sha256
 	$(BUILD_DIR)/gh-release create minishift/minishift-b2d-iso $(VERSION) master v$(VERSION)
+
+$(BIN_DIR)/minishift:
+	@echo "Downloading minishift binary..."
+	@mkdir -p $(BIN_DIR)
+	@cd $(BIN_DIR) && \
+	curl -LO --progress-bar https://github.com/minishift/minishift/releases/download/v1.0.0-beta.2/minishift-1.0.0-beta.2-linux-amd64.tgz && \
```

Comment:
Doesn't this smoke test now only works with linux because we are downloading only linux binary to do the testing? 

Smoke test suppose to run in every OS so might be adding OS specific flag would be nice.